Classes/Plugin/Calendar.php

```diff
@@ -49,6 +49,43 @@ class Calendar extends \Kitodo\Dlf\Common\AbstractPlugin
      */
     public function main($content, $conf)
     {
+        $this->init($conf);
+
+        // Set initial document (anchor or year file) if configured.
+        if (empty($this->piVars['id']) && !empty($this->conf['initialDocument'])) {
+            $this->piVars['id'] = $this->conf['initialDocument'];
+        }
+
+        // Load current document.
+        $this->loadDocument();
+        if ($this->doc === null) {
+            // Quit without doing anything if required variables are not set.
+            return $content;
+        }
+
+        $metadata = $this->doc->getTitledata($this->doc->cPid);
```

Comment:
We don't need to give `$this->doc->cPid` as parameter, since this is the default. Also, `$this->doc->cPid` is *write-only* and should not be used directly.
```suggestion
        $metadata = $this->doc->getTitledata();
```